lacci/lib/shoes/app.rb

```diff
@@ -247,30 +251,31 @@ def find_drawables_by(*specs)
         elsif spec.is_a?(Symbol) || spec.is_a?(String)
           s = spec.to_s
           case s[0]
-          when "$"
+          when '$'
             begin
               # I'm not finding a global_variable_get or similar...
               global_value = eval s
               drawables &= [global_value]
-            rescue
-              #raise Shoes::Errors::InvalidAttributeValueError, "Error getting global variable: #{spec.inspect}"
+            rescue StandardError
```

Comment:
Rescuing StandardError is the default. Any reason to mention it explicitly?

Reply:
that was an autoformatter rule for rescuing without specifying what we're rescuing. but if you don't say anything it reverts explicitly to the default. i have no opinion on this so i've reverted. and my autoformatter is now off